xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -333,6 +333,18 @@ namespace PVR
     */
    CDateTime FirstAired() const;
 
+   /*!
+    * @brief Get the premiere year of this event.
+    * @return The premiere year
+    */
+   int GetYear() const;
```

Comment:
Make both methods virtual in videoinfotag.h and 'override' here.

Reply:
No pvrguiinfo changes needed anymore, then.

Reply:
> 
> No pvrguiinfo changes needed anymore, then.

Thanks that is a much cleaner solution and resolved the return true issue as well